src/Network/Network.Test/SessionRecords/Commands.Network.Test.ScenarioTests.LoadBalancerTests/TestGatewayLoadBalancerProviderOnePool - Copy.json

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
I believe this file shouldn't be uploaded since it's named as 'xxx - Copy.json'.